topics/materials-science/tutorials/muspinsim/tutorial.md

```diff
@@ -0,0 +1,932 @@
+>    - {% icon param-file %} *"Configuration file containing simulation parameters"*: `muspinsim input file Low T` / `muspinsim input file High T` (Input dataset)
+> 2. Should generate the following output(s):
+>    - {% icon param-collection %} `Muspinsim output collection` (Output dataset)
+>    - {% icon param-file %} `muspinsim log for muspinsim input file Low T` / `muspinsim log for muspinsim input file High T` (Output dataset)
+>
+{: .hands_on}
+
+## MuSpinSim Plot
+
+We will use two files as inputs to the plot, however these are now displaying
+different data (low and high temperature).
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Plot](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim_plot/muspinsim_plot/3.5.1+galaxy0) %} with the following parameters:
+>    - *"Plot Title"*: `Temperature Example`
+>    - *"Label for x axis"*: `t`
+>    - *"Label for y axis"*: `Asymmetry`
+>    - *"Output File Type"*: `PNG`
+>    - *"Muspinsim File Series"*:
+>      - *"Muspinsim File Series 1"*:
+>        - {% icon param-file %} *"Muspinsim Experiment Data (.dat)"*: `Output dataset 'muspinsim_results' from step 3`
+>        - *"Line/Point Color"*: `Green`
+>        - *"Series Type"*: `Line`
+>        - *"Line type"*: `Solid`
+>        - *"Line width"*: `1.0`
+>      - *"Muspinsim File Series 2"*:
+>        - {% icon param-file %} *"Muspinsim Experiment Data (.dat)"*: `Output dataset 'muspinsim_results' from step 4`
+>        - *"Line/Point Color"*: `Blue`
+>        - *"Series Type"*: `Points`
+>        - *"Line type"*: `Square`
+>        - *"Line width"*: `2.0`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `MuSpinSim Plot on data 8 and data 7` (Output dataset)
+>
+{: .hands_on}
+
+![Resultant plot of the low and high temperature systems](../../images/temperature_example.png "The asymmetry oscillates differently in the low and high temperature regimes")
+
+The high temperature limit corresponds to the sum of two slightly out of phase
+cosinusoids, while the low temperature example is closer to a single wave. You
+can try experimenting with changing the temperature and re-running the
+simulation to see how it affects the result.
+
+# Workflow 6 - Dissipation
+
+Here we introduce another type of interaction for the muon: dissipation. Our
+muon is polarised in the $$x$$ direction, but the field applied is aligned with
+$$z$$. Under this perpendicular field, we expect the oscillations to decay
+according to the dissipation strength we provide.
+
+> <warning-title>Selecting the right dataset</warning-title>
+> If using a separate history for each workflow, switch to the history
+> containing only `dissipation_theory.dat` now. If sharing one history between
+> workflows, make sure that *"Theoretical Datapoints for Dissipation"* is
+> manually set to `dissipation_theory.dat`.
```

Comment:
The (perhaps too impicit) issue that @joelvdavies had, and that this looks to address, is the automatic dataset selection by Galaxy will grab the most recent `txt` file in your history when auto selecting (hence the history should **only** contain one dataset. Else you will need to **manually** select the right one. Having it in your history is a necessary requirement, but not sufficient to avoid trouble in all cases.

Will try and add to the warning to make it more explicit about what we're warning against here.

Reply:
Yeah, I tried to cover that in my suggestion - to me it didn't feel like as much of a problem in the manual process compared to the prebuilt workflow, since when you do the plotting manually you often have to change the automatically selected dataset anyway.